platform/authorized_api/user_groups/v1.yaml

```diff
@@ -0,0 +1,542 @@
+swagger: '2.0'
+
+info:
+  title: User Groups API
+  version: auto
+
+host: example.piwik.pro
+
+schemes: [https]
+
+paths:
+  /api/user-groups/v1:
+    post:
+      summary: User Group add
+      description: Adds user group with unique name
+      operationId: add_user_group_v1
+      produces:
+        - application/vnd.api+json
+      parameters:
+        - name: Add user group
+          in: body
+          description: Request data to add new user group
+          required: true
+          schema:
+            type: object
+            required:
+              - data
+            properties:
+              data:
+                $ref: '#/definitions/UserGroupEditable'
+      responses:
+        201:
+          description: Created
+        400:
+          description: Bad Request
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+    get:
+      summary: User Group list
+      description: Lists user groups
+      operationId: list_user_group_v1
+      produces:
+        - application/vnd.api+json
+      parameters:
+        - in: query
+          name: search
+          description: Search phrase - search for user group name and user group id
+          type: string
+          required: false
+        - in: query
+          name: sort
+          description: Sort type - can be reversed by adding dash before type e.g (-name)
+          type: string
+          default: name
+          required: false
+          enum:
+            - name
+            - created_at
+            - updated_at
+        - in: query
+          name: limit
+          description: Limits the number of returned user groups in response
```

Comment:
I remember we talked about generic descriptions for that kind of parameters e.g not `number returned user groups` but `number of entries`. Could we fix it here right or we leave it for another time? 

I think same thing could apply to `offset` as well.

Reply:
yeah, it's fixed on my [PPCEV-10201](https://clearcodehq.atlassian.net/browse/PPCDEV-10201): https://github.com/PiwikPRO/common-PPMS-AccessControl/blob/83cdac096bf148af1d13a4cd4b4b669ae9d6c334/docs/api/public/user-groups/v1.yaml#L489-L503